subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm_test.go

```diff
@@ -461,6 +566,33 @@ func testDeleteAllocatedPods(t *testing.T, h *HivedAlgorithm) {
 	}
 }
 
+func testReconfiguration(t *testing.T, sConfig *api.Config) {
+	h := NewHivedAlgorithm(sConfig)
+	for _, chains := range h.chains {
+		sortChains(chains)
+	}
+	testCasesThatShouldSucceed(t, h)
+
+	// case: physical cell not found
+	(*sConfig.PhysicalCluster).PhysicalCells[5].CellChildren[0].CellChildren[0].CellAddress = "0.0.3.100"
```

Comment:
Add case:

One large PC breaks to multiple small PCs

Reply:
that is the case below "inconsistent physical and virtual cell hierarchies"

---
In reply to: [339981836](https://github.com/microsoft/pai/pull/3768#discussion_r339981836) [](ancestors = 339981836)

Reply:
The comment "inconsistent physical and virtual cell hierarchies" is too general

---
In reply to: [341835370](https://github.com/microsoft/pai/pull/3768#discussion_r341835370) [](ancestors = 341835370,339981836)